lastpasslib/lastpasslib.py

```diff
@@ -653,7 +654,19 @@ def get_passwords(self):
             A list of password type secrets.
 
         """
-        return self.get_secrets(filter_='Password')
+        return self.get_secrets(filter_="Password")
+
+    def delete_secret(self, name):
+        """Deletes a secret from the vault.
+
+        Returns:
+            bool: True on success, raises an HTTPError exception otherwise.
+
+        """
+        secret = self.get_secret_by_name(name)
```

Comment:
This will raise an exception if multiple secrets exist with the same name, this should be in the documentation to expose that functionality to the user.